tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/WebRequestTests.cs

```diff
@@ -37,26 +37,28 @@ public void SubmitsTraces()
             Output.WriteLine($"Assigning port {httpPort} for the httpPort.");
 
             using (var agent = EnvironmentHelper.GetMockAgent())
-            using (ProcessResult processResult = RunSampleAndWaitForExit(agent.Port, arguments: $"Port={httpPort}"))
             {
-                var spans = agent.WaitForSpans(expectedSpanCount, operationName: expectedOperationName).OrderBy(s => s.Start);
-                spans.Should().HaveCount(expectedSpanCount);
-
-                foreach (var span in spans)
+                using (ProcessResult processResult = RunSampleAndWaitForExit(agent, arguments: $"Port={httpPort}"))
```

Comment:
You're like my nemesis, adding extra indents. I've been going through doing

```diff
- using (var agent = EnvironmentHelper.GetMockAgent())
- {
-     someThing();
+ using var agent = EnvironmentHelper.GetMockAgent();
+ someThing();
```

And here you are going the other way 🤣

Reply:
Hahaha, sorry.. I *really like* brackets, okay?
I'll remove it though, this isn't needed.

Reply:
Removed